www/package.json

```diff
@@ -65,7 +65,10 @@
     "typeface-spectral": "^0.0.54",
     "typography": "^0.16.6",
     "typography-breakpoint-constants": "^0.15.10",
-    "typography-plugin-code": "^0.16.11"
+    "typography-plugin-code": "^0.16.11",
+    "zipkin": "^0.14.0",
+    "zipkin-javascript-opentracing": "^1.6.0",
+    "zipkin-transport-http": "^0.14.0"
```

Comment:
Did you mean to add zipkin dependencies here?

Reply:
Yeah, they allow us to run tracing on the www site. We can drop them if you prefer, though.

Reply:
Nah all good, let's leave them in 👍